integration.json

```diff
@@ -1,7 +1,7 @@
 {
   "build-plan": "github.com/paketo-community/build-plan",
+  "node-engine": "github.com/paketo-buildpacks/node-engine",
   "builders": [
-    "index.docker.io/paketocommunity/builder:ubi-buildpackless-base",
-    "index.docker.io/paketocommunity/builder-ubi-buildpackless-base:latest"
+    "index.docker.io/paketocommunity/builder-ubi-buildpackless-base:0.0.3"
```

Comment:
@pacostas @mhdawson Is there a reason why we don't want to use `index.docker.io/paketocommunity/builder-ubi-buildpackless-base:latest`?

Reply:
This is pinned to version 0.0.3 temporarily due to the lifecycle of the `latest`  `builder-ubi-buildpackless-base`  for some reason ( @mhdawson knows more) is earlier and does not include support for the extension.

Reply:
@pacostas says the issue is that after 0.0.3 was published the automation immediately ran an reverted the lifecycle to an earlier version so later version will not currently work. Once a new version of the lifecycle is published the automation should then publish a version with a new enough lifecycle and we will unpin this.

Reply:
Makes sense, thanks. Given that lifecycle v0.17.0 has been released, can we relax this constraint back to `latest` now?